bin/http-server

```diff
@@ -40,6 +40,7 @@ if (argv.h || argv.help) {
     '',
     '  -r --robots  Respond to /robots.txt [User-agent: *\\nDisallow: /]',
     '  --no-dotfiles  Do not show dotfiles',
+    '  -V --superverbose    super verbose mode',
```

Comment:
Could we move this right next to `--silent` above, since they're both about logging?